src/Schema/Directives/CanDirective.php

```diff
@@ -9,12 +9,14 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\ModelNotFoundException;
 use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Auth;
```

Comment:
We try to not use facades: https://github.com/nuwave/lighthouse/blob/master/CONTRIBUTING.md#laravel-feature-usage

Reply:
Thanks for the advice !